Countries Info/Managers/ImageManager.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  ImageManager.swift
+//  Countries Info
+//
+//  Created by Oleh Stasiv on 03.12.2020.
+//
+
+import Foundation
+
+class ImageManager {
+    static let shared = ImageManager()
+    
+    private init() {}
+    
+    func fetchImage(url: String?) -> Data? {
+        guard let stringURL = url else { return nil }
+        guard let imageURL = URL(string: stringURL) else { return nil}
+        guard let imageData = try? Data(contentsOf: imageURL) else { return nil }
```

Comment:
This will potentially block the UI, for example scrolling may be happening not smoothly.
As a first thought, we may use some completion handler and pass the result back, when it will be fetched successfully.